package.json

```diff
@@ -18,28 +18,36 @@
     "eyeglass-module"
   ],
   "engines": {
-    "node": ">=8"
+    "node": ">=10"
   },
   "dependencies": {
-    "chalk": "^2.4.2",
-    "css": "^2.2.1",
-    "lodash.find": "^4.6.0",
-    "lodash.foreach": "^4.5.0",
-    "lodash.last": "^3.0.0"
+    "chalk": "^4.1.0",
+    "css": "^3.0.0",
+    "lodash": "^4.17.19"
+  },
+  "peerDependencies": {
+    "sass": ">=1.23.0"
```

Comment:
@mirisuzanne I'm not really sure what version of `sass` we require, but this seemed like a bare minimum. 🤷

Reply:
Yeah, that's a good guess. I don't think we're using anything terribly advanced, so probably safe